Harvest.Core/Services/AggieEnterpriseService.cs

```diff
@@ -95,7 +95,11 @@ public async Task<AccountValidationModel> IsAccountValid(string financialSegment
                 }
 
                 rtValue.GlSegments = FinancialChartValidation.GetGlSegments(financialSegmentString);
-
+                
+                rtValue.CoaDescription = $"{data.GlValidateChartstring.SegmentNames.DepartmentName} - {data.GlValidateChartstring.SegmentNames.FundName}";

```

Comment:
Are some department names kind of wordy? Should this be wrapped in some form of safe-truncate?

Reply:
Seems fine
![image](https://user-images.githubusercontent.com/469010/220724319-d9f174f9-117d-43a6-8a6e-48fa14c340c6.png)

![image](https://user-images.githubusercontent.com/469010/220724421-779bd2a2-65fa-44e0-9a3a-b3955fbc8a67.png)